patches/chrome-browser-extensions-install_verifier.cc.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/chrome/browser/extensions/install_verifier.cc b/chrome/browser/extensions/install_verifier.cc
+index 23b122e8627a262388927734317a1582b73542ff..319c9b5b6c76f978c1913cd6472f42139f8cb619 100644
+--- a/chrome/browser/extensions/install_verifier.cc
++++ b/chrome/browser/extensions/install_verifier.cc
+@@ -378,6 +378,7 @@ bool InstallVerifier::MustRemainDisabled(const Extension* extension,
+                                          disable_reason::DisableReason* reason,
+                                          base::string16* error) const {
+   CHECK(extension);
++  ReturnFalseIfExtensionShouldSkipSignatureChecking(extension->id())
```

Comment:
This happens because for `Media Router` the signature fetch and check happens out of order. This is likely a chromium bug, but not seen because chromium uses ENFORCE instead of ENFORCE_STRICT by default.(https://cs.chromium.org/chromium/src/chrome/browser/extensions/chrome_content_verifier_delegate.cc?l=110) 

But, `ENFORCE_STRICT` (https://cs.chromium.org/chromium/src/extensions/browser/content_verifier_delegate.h?g=0&l=42) is better for security. 

We have multiple options here:

1. Use `ENFORCE` over `ENFORCE_STRICT` similar to chromium.
2. SkipSignatureVerification for MediaRouter as seen in this PR.
3. RedoSignature check after the fetch is complete.

Thoughts? @bridiver  @diracdeltas @tomlowenthal  

cc: @bsclifton @simonhong

Reply:
Option 2 or 3 sound great to me- I'd vote for option 2 as my choice 😄

Reply:
i prefer 3 since Google Chrome uses ENFORCE_STRICT (https://cs.chromium.org/chromium/src/chrome/browser/extensions/chrome_content_verifier_delegate.cc?l=68); so IIUC, 3 is the only option that gives us parity with Google Chrome security-wise.

Reply:
@diracdeltas - `ENFORCE_STRICT` is only used when field trials are enabled, I don't think Chrome has field trials enabled for release builds. Let me know if I'm missing something.

Reply:
@diracdeltas I thought it was the other way around? Chromium uses `ENFORCE` so my preference is to follow that and not patch here

Reply:
@diracdeltas @bridiver - `ENFORCE_STRICT` works well for the extensions I have tested, but I understand the concern that we might run into web-compat issues with an experimental feature. 

I have a patch for `3` but, I'm okay with `1` to avoid any web-compat risk we might run into.